server/email_templates/jira_account_denied_pmc.txt

```diff
@@ -1,12 +1,15 @@
 Jira account request denied by {approver}: {userid}
 --
-Hi there,
+Hi, there.
+
 The request for a new Jira account, {userid}, was denied by {approver}.
-The request will automatically be removed from the database in 24 hour from now.
+The request will automatically be removed from the database 24 hours from now.
 
-If this decision appears to have been a mistake, you can still revisit the application
+If this decision appears to have been a mistake, you can revisit the application
 and approve it by visiting the original review URL from the initial account request
-email within 24 hours from now.
+email within the next 24 hours.
+
+Note: You must wait 24 hours before submitting a new request for this Jira account.
```

Comment:
this 24h notice is not intended for the PMC, but for the user, so shouldn't be in this template

Reply:
removed